doc/source/whatsnew/v1.4.0.rst

```diff
@@ -608,6 +608,7 @@ Datetimelike
 - Bug in adding a ``np.timedelta64`` object to a :class:`BusinessDay` or :class:`CustomBusinessDay` object incorrectly raising (:issue:`44532`)
 - Bug in :meth:`Index.insert` for inserting ``np.datetime64``, ``np.timedelta64`` or ``tuple`` into :class:`Index` with ``dtype='object'`` with negative loc adding ``None`` and replacing existing value (:issue:`44509`)
 - Bug in :meth:`Series.mode` with ``DatetimeTZDtype`` incorrectly returning timezone-naive and ``PeriodDtype`` incorrectly raising (:issue:`41927`)
+- Bug in :class:`DateOffset` with ``nanoseconds`` component does not affect :class:`Timestamp` when adding them. (:issue:`43968`)
```

Comment:
Bug in :class:`DateOffset` addition with :class:`Timestamp` where ``offset.nanoseconds`` would not be include din the result

Reply:
Done, thanks~